WordPress/src/main/res/values/strings.xml

```diff
@@ -1142,6 +1142,7 @@
     <string name="feedback_form_attachment_already_added">Attachment already added</string>
     <string name="feedback_form_unable_to_create_tempfile">Unable to create temporary file</string>
     <string name="feedback_form_add_attachments">Add attachments</string>
+    <string name="feedback_form_note">If you need support, please get in touch using the \"Help &amp; Support\" screen</string>
```

Comment:
This feels like sentence that should have closing punctuation. Feel free to disagree and ignore.

```suggestion
    <string name="feedback_form_note">If you need support, please get in touch using the \"Help &amp; Support\" screen.</string>
```

Reply:
@dcalhoun That does seem like it needs a period, but I try to follow the Material guidelines about punctuation.

![Screenshot 2024-10-24 at 10 38 03 AM](https://github.com/user-attachments/assets/a5130776-f919-4a75-bb33-c9095ae2fc8d)